android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -2,6 +2,7 @@
 <network-security-config>
     <domain-config cleartextTrafficPermitted="true">
         <domain includeSubdomains="true">10.0.2.2</domain>
+        <domain includeSubdomains="true">192.168.0.168</domain>
```

Comment:
Do we need this? Did I add it?

Reply:
Ooops, this shouldn't have been comitted. It was mainly to connect to Metro via wifi, but Oliver did another PR wher this isn't needed anymore.

Reply:
Done

Reply:
Done